src/refactor_nrepl/core.clj

```diff
@@ -65,18 +65,28 @@
   Should generally always be true, except in a few refactor-nrepl's unit tests."
   true)
 
+(def ^:private resources-dir-pattern
+  (re-pattern (str "resources" File/separator "?$")))
+
+(def ^:private target-dir-pattern
+  (re-pattern (str "target" File/separator "?$")))
+
+(defn irrelevant-dir?
+  "Does `f` look like a directory without files that should be analyzed/refactored?"
+  [^File f]
+  (let [s (-> f .toString)]
+    (boolean (or (if *skip-resources?*
+                   (re-find resources-dir-pattern s)
+                   false)
+                 (re-find target-dir-pattern s)
+                 (-> s (.contains ".gitlibs"))))))
```

Comment:
I think in Clojure 1.8 there was a matching function in `clojure.string`.

Reply:
Can use, not against it but looks like I'm not adding another commit anyway

In general in projects with zero reflection warnings (an increasing reality 😇) one can afford to call these fearlessly and with full tooling support e.g. Compliment